### PR TITLE
KC-5637 Breaking out automation, analytics, and integration into separate projects

### PR DESCRIPTION
**Pull request by acordova**

### CODE REVIEW

File: analytics/java/src/main/java/com/koverse/examples/analytics/JavaWordCounter.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Koverse, Inc.
+ * Copyright 2018 Koverse, Inc.
```
Comment (bmcrea) [inline analytics/java/src/main/java/com/koverse/examples/analytics/JavaWordCounter.java]: Actually it should be 2019 now.

File: automation/pom.xml [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-shade-plugin</artifactId>
+                    <version>2.3</version>
+                </plugin>
+                <plugin>
+                    <groupId>org.apache.rat</groupId>
+                    <artifactId>apache-rat-plugin</artifactId>
+                    <version>0.11</version>
+                </plugin>
+                <!-- <plugin>
```
Comment (bmcrea) [inline automation/pom.xml]: Do we need this in the examples repo?

Comment (acordova) [reply automation/pom.xml]: I think we can reinstate these checks once we get all the code moved over. I didn't want these to hold up the migration of all this code

File: analytics/java/src/main/java/com/koverse/examples/analytics/JavaWordCounter.java (hunk 1 above)
Comment (acordova) [reply analytics/java/src/main/java/com/koverse/examples/analytics/JavaWordCounter.java]: Not sure why I updated that. I think we can leave the original dates in place

**Pull request merged**